contracts/modules/cover/Gateway.sol

```diff
@@ -74,6 +76,7 @@ contract Gateway is MasterAware {
     quotationData = QuotationData(master.getLatestAddress("QD"));
     claimsData = ClaimsData(master.getLatestAddress("CD"));
     claims = Claims(master.getLatestAddress("CL"));
+    incidents = Incidents(master.getLatestAddress("IC"));
```

Comment:
Remove the DAI if condition below, it's already initialized.

Reply:
Removing it seems to break the test since DAI isn't already initialised and thus resulting in a revert: `Cover: unknown asset`. On that note, I'd rename Cover revert reason prefix to Gateway and revert those changes.